### ISSUE TITLE
Delete / Copy to Clipboard options are hard to find in Data tab

### ISSUE THREAD

**Issue by mbookman** [DataBiosphere/saturn-ui#1225]
I have a table of 1895 samples with about 40 columns.
In the Data tab/Tables item, it was very hard to find the Delete (trashcan) icon.
I had to scroll a lot to get to the far right. If I didn't know the feature was there, I never would have found it.
Can this option be moved to the top bar where "Download Selected TSV" and "Copy to Clipboard" are?

**Comment by katevoss** [comment]
Hi @mbookman, thanks for the feedback! I have reported this issue to Terra's Feature Request page to continue the conversation over there: https://broadinstitute.zendesk.com/hc/en-us/community/posts/360042595392-Delete-Data-Copy-Data-options-are-hard-to-find-in-Data-tab

**Comment by mbookman** [comment]
Thanks Kate.

I was looking at this again and noticed interesting behavior that I hadn't observed before.
I proposed a new way to implement this, which arguably is a "feature request", but I think that this may just be a bug in that the UI isn't working the way it was intended. If the bug were fixed I would retract the suggested alternate UI:

Here's the discovery:

I have an entity (table) with many columns.

When I click on any of the check boxes on the left hand side of the data table, I can't see the trash can or the clipboard copy icon that appear because they are floating all the way off to the right side of the frame.

If I scroll all the way to the right, I see these icons.
If I scroll back to the left, the icons stay on the screen until I scroll *all the way* back to the left.

This doesn't actually require a super-wide entity to reproduce the problem. Just narrowing the browser window makes it easy to reproduce.

If the icon positioning issue were fixed, then I think that's all that is needed. I had not realized yesterday that this icons were "floating" and were intended to be at the right of the *visible* portion of the frame, rather than on the right hand side of the frame.

**Comment by zarsky-broad** [comment]
For the moment, we had decided on a minimum width for the site of 1200 px, under which it scrolls. We can discuss if this is too high.

**Comment by bradtaylor** [comment]
I agree it is inconsistent to have some controls above the table and some off to the right. Our latest mocks have controls on the data table centralized into a bottom row of buttons in the table area. Those mocks are still being refined. I expect we'll move to this implementation early in Q2. I've added the 2019-Q2 label. I'm going to move this into a corresponding (rough) position on the backlog

**Comment by mbookman** [comment]
Sounds great. Thanks Brad!
